server/src/test/java/com/adobe/testing/s3mock/domain/FileStoreTest.java

```diff
@@ -675,13 +678,41 @@ public void getObjectsForNullPrefix() throws Exception {
   public void getObjectsForPartialParentDirectory() throws Exception {
     fileStore.createBucket(TEST_BUCKET_NAME);
     fileStore
-        .putS3Object(TEST_BUCKET_NAME, "a/bee/c", "text/plain",
+        .putS3Object(TEST_BUCKET_NAME, "a/bee/c", TEXT_PLAIN,
             new FileInputStream(new File(TEST_FILE_PATH)),
             false);
     final List<S3Object> result = fileStore.getS3Objects(TEST_BUCKET_NAME, "a/b");
     assertThat(result, is(empty()));
   }
 
+  @Test
+  public void multipartUploadBrokenByLexicalPartSorting() throws IOException {
```

Comment:
`multipartUploadPartsAreSortedNumerically` to state what's expected to work and not what the issue was that had been fixed?